pyccl/halos/concentration/ishiyama21.py

```diff
@@ -27,108 +27,36 @@ class ConcentrationIshiyama21(Concentration):
             method. Otherwise, use the concentration found with profile
             fitting. The default is False.
     """
-    __repr_attrs__ = ("mdef", "relaxed", "Vmax",)
+    __repr_attrs__ = ("mass_def", "relaxed", "Vmax",)
     name = 'Ishiyama21'
 
-    def __init__(self, mdef=None, relaxed=False, Vmax=False):
+    @warn_api(pairs=[("mdef", "mass_def")])
+    def __init__(self, *, mass_def="500c",
```

Comment:
Why make 500c the default? It's cool that they include it, but 200c or 200m are in general more common choices, so that'd minimize surprises

Reply:
That's how we had originally implemented it in #895 . Do you want to change that?